projects/plugins/jetpack/class.jetpack.php

```diff
@@ -713,15 +713,6 @@ private function __construct() {
 			3
 		);
 
-		/**
-		 * Prepare Gutenberg Editor functionality
-		 */
-		require_once JETPACK__PLUGIN_DIR . 'class.jetpack-gutenberg.php';
-		add_action( 'plugins_loaded', array( 'Jetpack_Gutenberg', 'init' ) );
-		add_action( 'plugins_loaded', array( 'Jetpack_Gutenberg', 'load_independent_blocks' ) );
-		add_action( 'plugins_loaded', array( 'Jetpack_Gutenberg', 'load_block_editor_extensions' ), 9 );
-		add_action( 'enqueue_block_editor_assets', array( 'Jetpack_Gutenberg', 'enqueue_block_editor_assets' ) );
```

Comment:
I'm sorry, I know I mentioned this to you. But I realize 2 things now:

1. The class is used later in the file:
https://github.com/Automattic/jetpack/blob/caf0fcbe7b2953f470e2a3913450b426c7aae3d3/projects/plugins/jetpack/class.jetpack.php#L1075
So maybe the require remains needed?
2. Some sites will have the module auto-activation turned off, and thus will lose the blocks when updating the plugin, so I wonder if we should take a different approach here. Maybe add the new `blocks.php` file to this list so it's always loaded:
https://github.com/Automattic/jetpack/blob/29e8f84bf5319caffdc6ba27321616e9358826a3/projects/plugins/jetpack/modules/module-extras.php#L15

And then handle the toggle differently, via an option that would short-circuit things here:
https://github.com/Automattic/jetpack/blob/759595d74eb8abb01044877b2cd30a56743246ac/projects/plugins/jetpack/class.jetpack-gutenberg.php#L461-L474

Reply:
Good idea, used a regular option for the disabling, rather than a Jetpack_Option.